functions/models/src/fhir/fhirQuestionnaireResponse.ts

```diff
@@ -20,37 +20,76 @@ import { optionalish } from '../helpers/optionalish.js'
 import { SchemaConverter } from '../helpers/schemaConverter.js'
 import { type SymptomQuestionnaireResponse } from '../types/symptomQuestionnaireResponse.js'
 
-export const fhirQuestionnaireResponseItemConverter = new Lazy(
-  () =>
-    new SchemaConverter({
-      schema: z.object({
-        answer: optionalish(
-          z
-            .object({
-              valueCoding: optionalish(
-                z.lazy(() => fhirCodingConverter.value.schema),
-              ),
-            })
-            .array(),
-        ),
-        linkId: optionalish(z.string()),
-      }),
-      encode: (object) => ({
-        answer:
-          object.answer?.flatMap((value) => ({
-            valueCoding:
-              value.valueCoding ?
-                fhirCodingConverter.value.encode(value.valueCoding)
-              : null,
-          })) ?? null,
-        linkId: object.linkId ?? null,
-      }),
-    }),
-)
+const fhirQuestionnaireResponseItemBaseConverter = new SchemaConverter({
```

Comment:
It would be amazing if we can add one unit test with an Android Questionnaire output to verify that this is working in the future and we don't have any regressions on this. Maybe @eldcn or @Basler182 can provide a response that we currently get from the app 👍

Reply:
Well, technically we have the firebase content, but it doesn't seem to be super easy to get a JSON object of that, so a JSON-formatted response would be super helpful!

Reply:
Sounds good; feel free to each out to @Basler182 & @eldcn if we can easily get a copy; otherwise we might need to manually pull it from Firebase.